contracts/snapshots/MerkleSnapshot.sol

```diff
@@ -0,0 +1,25 @@
+pragma solidity ^0.5.11;
+
+import "../zeppelin//MerkleProof.sol";
+import "../ManagerProxyTarget.sol";
+
+
+contract MerkleSnapshot is ManagerProxyTarget{
```

Comment:
Could this contract inherit from `Manager` instead of `ManagerProxyTarget`?

The contract would only need to inherit from `ManagerProxyTarget` if it needs to be upgradeable as a part of a proxy setup (so that the proxy and the implementation share the same exact storage layout). Given that this contract is relatively simple and the state would be easily migrated if needed, I don't think it needs to be upgradeable as a part of a proxy setup. WDYT?

Reply:
fixed in 2839cec